openssl/src/asn1.rs

```diff
@@ -75,6 +75,28 @@ impl Asn1StringRef {
     }
 }
 
+type_!(Asn1Integer, Asn1IntegerRef, ffi::ASN1_INTEGER, ffi::ASN1_INTEGER_free);
+
+impl Asn1IntegerRef {
+    pub fn get(&self) -> i64 {
+        unsafe {
+            return ::ffi::ASN1_INTEGER_get(self.as_ptr());
+        }
+    }
+
+    pub fn set(&self, value: i64) -> Result<(), ErrorStack>
```

Comment:
Hmm, taking an i64 is a bit unfortunate since that's lossy on 32 bit platforms.

@alexcrichton any ideas what the right approach is here? We could take an i32 here, but then it's asymmetric with get...